doc/source/myrocks/variables.rst

```diff
@@ -2521,6 +2554,17 @@ Specifies whether to generate trace output in the log
 for each call to ``SstFileWriter``.
 Disabled by default.
 
+.. variable:: rocksdb_track_and_verify_wals_in_manifest
+
+  :version 5.7.35-38: Implemented
+  :cli: ``--rocksdb-track-and-verify-wals-in-manifest``
+  :dyn: No
+  :scope: Global
+  :vartype: Boolean
+  :default: ``ON``
+
+DBOptions::track_and_verify_wals_in_manifest for RocksDB
```

Comment:
My bad, I should have added this in the ticket.

It could be positive describing the option:

If true, the log numbers and sizes of the synced WALs are tracked
in MANIFEST, then during DB recovery, if a synced WAL is missing
from disk, or the WAL's size does not match the recorded size in
MANIFEST, an error will be reported and the recovery will be aborted.

Note that this option does not work with secondary instance.

Reply:
Added.